modules/enterprise/server/installer/src/main/java/org/rhq/enterprise/server/installer/ServerInstallUtil.java

```diff
@@ -261,6 +262,11 @@ public static void configureDeploymentScanner(ModelControllerClient mcc) throws
         LOG.info("Deployment scanner turned off.");
     }
 
+    public static void configureSystemProperties(ModelControllerClient mcc) throws Exception {
+        CoreJBossASClient client = new CoreJBossASClient(mcc);
+        client.setSystemProperty("jackson.deserialization.whitelist.packages", "org.rhq.enterprise.server.rest.domain");
```

Comment:
This is technically only something that works in the JON-release (master branch has different jackson version). Did you test with the master that it does not break anything?

Reply:
I didn't check the logs. I installed and started RHQ and checked that everything was in there.
Will double check that nothing seems broken.

Reply:
Tested and it doesn't seem to break anything on master branch.